charts/k8ssandra/Chart.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
     repository: file://../medusa-operator
     condition: medusa.enabled
   - name: k8ssandra-common
-    version: 0.28.4
+    version: 0.28.5
```

Comment:
All the other charts depend on k8ssandra-common as well. You will need to update the dependencies in other charts and then bump their chart versions.